mujoco_warp/testspeed.py

```diff
@@ -151,6 +155,14 @@ def _main(argv: Sequence[str]):
         m.opt.integrator = k
 
   m.opt.ls_parallel = _LS_PARALLEL.value
+  if _BROADPHASE.value is not None:
+    m.opt.broadphase = _BROADPHASE.value
+  if _BROADPHASE_FILTER.value is not None:
+    m.opt.broadphase_filter = _BROADPHASE_FILTER.value
+
```

Comment:
nit: either delete this newline or add newlines above the other conditionals.

Reply:
removed graph_conditional